<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPresentacion</class>
 <widget class="QDialog" name="DialogPresentacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnPreview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>210</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDiapositiva">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Diapositiva actual</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>578</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnNuevo">
      <property name="text">
       <string>Nueva diapositiva</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="text">
       <string>ELiminar diapositiva</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGuardar">
      <property name="text">
       <string>Guardar diapositiva</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>301</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnTitulo">
      <property name="text">
       <string>Titulo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCompleta">
      <property name="text">
       <string>Completa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDoble">
      <property name="text">
       <string>Doble</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>401</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lblTitulo">
      <property name="text">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtTitulo">
      <property name="placeholderText">
       <string>Ingrese un título</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>401</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblImagen1">
      <property name="text">
       <string>Imagen 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="edtImagen1">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Path primera imagen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="btnImagen1">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>401</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="lblImagen2">
      <property name="text">
       <string>Imagen 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="edtImagen2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Path segunda imagen</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="btnImagen2">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="lblContenido">
      <property name="text">
       <string>Contenido</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="edtContenido">
      <property name="placeholderText">
       <string>Ingrese su contenido</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>411</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="btnGuardar_2">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGrafico">
      <property name="text">
       <string>Gráfico</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPDF">
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
